fastify.js

```diff
@@ -237,17 +237,17 @@ function build (options) {
     fn(this.req, this.res, cb)
   }
 
-  function override (instance, fn, opts) {
+  function override (old, fn, opts) {
     if (fn[Symbol.for('skip-override')]) {
-      return instance
+      return old
     }
 
-    const middlewares = Object.assign([], instance._middlewares)
-    instance = Object.create(instance)
+    const middlewares = Object.assign([], old._middlewares)
+    const instance = Object.create(old)
     instance._Reply = Reply.buildReply(instance._Reply)
     instance._Request = Request.buildRequest(instance._Request)
     instance._contentTypeParser = ContentTypeParser.buildContentTypeParser(instance._contentTypeParser)
-    instance._hooks = Hooks.buildHooks(instance._hooks)
+    instance._hooks = Hooks.buildHooks(instance._hooks, instance)
```

Comment:
`instance` is not more needed, as well as [`._fastify`](https://github.com/fastify/fastify/blob/ac1eec2bf60357a88d7e17ba577d7fde58db784f/lib/hooks.js#L14) inside `hooks.js`

Reply:
done